src/node/http.ts

```diff
@@ -69,6 +71,39 @@ export const authenticated = (req: express.Request): boolean => {
           ? safeCompare(req.cookies.key, req.args["hashed-password"])
           : req.args.password && safeCompare(req.cookies.key, hash(req.args.password)))
       )
+    case AuthType.Openid:
+      if (req.oidc.isAuthenticated()) {
+        console.debug("User is authenticated using OpenID Connect\n", req.oidc.user)
```

Comment:
Thank you so much for your PR! Only nit is that we'd prefer you use `logger.debug` instead of `console.debug` for consistency with the rest of our codebase.

For example: `logger.debug("User authenticated with OpenID Connect", field("user", req.oidc.user))`

Sorry about the `console.debug` suggestion earlier :sweat_smile:

Reply:
haha, no worries, consistency is king! I'll get it updated and pushed at some point today.

Reply:
The updates have been made. Let me know if there's anything else that needs to be tweaked.

Reply:
I pushed one last change to update the logger.debug calls to use field() instead of whatever it is I did.